services/kommander/0.4.0/defaults/cm.yaml

```
@@ -82,6 +82,28 @@ data:
           traefik.ingress.kubernetes.io/router.middlewares: "${workspaceNamespace}-stripprefixes@kubernetescrd,${workspaceNamespace}-forwardauth@kubernetescrd"
         path: /dkp/kommander/dashboard
         graphqlPath: /dkp/kommander/dashboard/graphql
+    capimate:
+      enabled: true
+      replicas: 2
+      image:
+        repository: mesosphere/capimate
+        pullPolicy: IfNotPresent
+        tag: v0.0.0-dev.0
+      env: []
+      extraArgs: []
+      resources:
+        limits:
+          cpu: 500m
+          memory: 512Mi
+        requests:
+          cpu: 100m
+          memory: 128Mi
+      securityContext:
+        runAsUser: 65532
+      service:
+        annotations: {}
+        port: 8443
+        type: ClusterIP
```

```suggestion
    capimate:
      image:
        tag: v0.0.0-dev.0
```
we should just override the tag here, the rest we can stick with the default chart values